Controllers/ProductsController.cs

```diff
@@ -193,6 +193,21 @@ public IActionResult Create([FromRoute]int id,ProductCreate product)
             return View(model);
         }
 
+        [HttpPost]
+        public IActionResult GetSubTypes(int id, [FromBody] ProductSubTypeForm productCreate)
+        {
+            ProductEdit model = new ProductEdit(context);
+
+            model.CurrentProduct = new Product();
+
+            model.CurrentProduct.Name = productCreate.Name;
+            model.CurrentProduct.Description = productCreate.Description;
+            model.CurrentProduct.Price = (decimal)productCreate.Price * 10;
+            model.CurrentProduct.ProductTypeId = id;
+
+            return View(model);

```

Comment:
Route does not reload from this method being fired by the onchange so the return does not seem to work.